Examples/Common/Keys.m

```diff
@@ -5,18 +5,22 @@ @implementation Keys
 + (NSString const *) universeName {
     // Set your universe name here - https://support.backtrace.io/hc/en-us/articles/360040516451-What-is-a-submission-url-
     // e.g.: yourteam
-    return @"yourteam";
+    return @"melektest";
```

Comment:
this shouldn't be here

Reply:
that and other things too!
I'll cleanup soon

Reply:
Yeah but if we're announcing to have something to test and share we need to be sure something like that will be cleaned up.